android/TritonClient/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Just a question, Should you add .idea also to your .gitignore?
<img width="458" alt="Screen Shot 2021-07-01 at 9 32 53 AM" src="https://user-images.githubusercontent.com/24863504/124158878-5186ea80-da4f-11eb-8d60-5c14778f2030.png">

Reply:
You're comparing top-level .gitignore to the app-level version. Take a look at `android/TritonClient/.gitignore`, which has specific .idea entries.

Reply:
Aha, Ok thanks for the explanation.